examples/timeseries.md

```diff
@@ -3,13 +3,15 @@ title: Timeseries
 author: Tom Augspurger
 ---
 
+# Timeseries
```

Comment:
This also improves the [look of the document from GitHub](https://github.com/kiwi0fruit/stitch/blob/master/examples/timeseries.md). And this time the file is still a valid Stitch document.